code/addons/docs/src/preset.ts

```diff
@@ -52,15 +54,36 @@ async function webpack(
     mdxPluginOptions = {},
   } = options;
 
+  const rehypeFencedCode = function () {
+    return (tree: Root) => {
+      visit(tree, 'element', (node: any) => {
+        const metastring = node.data?.meta;
+        if (node.tagName === 'code' && metastring) {
+          node.properties = {
```

Comment:
I think CI is failing with a lint error here.

Reply:
I will take a look shortly

Reply:
done, thanks for the lookout